### ISSUE TITLE
Could not compile `triple_accel`

### ISSUE THREAD

**Issue by TianyiShi2001** [Daniel-Liu-c0deb0t/triple_accel#3]
System information:

```
Linux tianyi-optiplex3070 5.6.15-1-MANJARO #1 SMP PREEMPT Wed May 27 20:38:56 UTC 2020 x86_64 GNU/Linux
```
Cargo version:

```
cargo 1.48.0-nightly (875e01232 2020-09-08)
```
Error message:

```
error: attribute should be applied to a function
    --> /home/tianyi/.cargo/registry/src/mirrors.ustc.edu.cn-61ef6e0cd06fb9b8/triple_accel-0.3.2/src/jewel.rs:1500:21
     |
1499 | /                 unsafe {
1500 | |                     #![target_feature(enable = "sse4.1")]
     | |                     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
1501 | |                     write!(f, "[")?;
1502 | |
...    |
1528 | |                     write!(f, "]")
1529 | |                 }
     | |_________________- not a function
...
1540 |   create_sse_nx16x8!(Sse16x16x8, 16);
     |   ----------------------------------- in this macro invocation
     |
     = note: this error originates in a macro (in Nightly builds, run with -Z macro-backtrace for more info)

error: aborting due to 13 previous errors

error: could not compile `triple_accel`

Caused by:
  process didn't exit successfully: `rustc --crate-name triple_accel --edition=2018 /home/tianyi/.cargo/registry/src/mirrors.ustc.edu.cn-61ef6e0cd06fb9b8/triple_accel-0.3.2/src/lib.rs --error-format=json --json=diagnostic-rendered-ansi --crate-type lib --emit=dep-info,metadata,link -C embed-bitcode=no -C debuginfo=2 --cfg 'feature="default"' --cfg 'feature="jewel-16bit"' --cfg 'feature="jewel-32bit"' --cfg 'feature="jewel-8bit"' --cfg 'feature="jewel-avx"' --cfg 'feature="jewel-sse"' -C metadata=ea803cb5cc9b7c41 -C extra-filename=-ea803cb5cc9b7c41 --out-dir /home/tianyi/a/target/debug/deps -L dependency=/home/tianyi/a/target/debug/deps --cap-lints allow` (exit code: 1)
```

(multiples errors of the same kind, all saying `attribute should be applied to a function`)

**Comment by Daniel-Liu-c0deb0t** [comment]
Just reproduced this. Seems to be an issue only in the nightly version.

**Comment by Daniel-Liu-c0deb0t** [comment]
This should be fixed in the latest version (v0.3.3) of the crate.
